src/Pages/BookPage.js

```diff
@@ -0,0 +1,166 @@
+    const image = getDefault(book.get('image'), [{ url: TCK_LOGO }]);
+    const imageURL = image[0].url;
+    const readAloudURL = getDefault(book.get('read_aloud_link'), null);
+    const bookshopURL = getDefault(book.get('bookshop_link'), null);
+    const educatorURLs = getDefault(book.get('educator_guide_links'), []);
+
+    const synopsis = (
+      <div style={{ display: 'flex', flexDirection: 'row' }}>
+        <CardMedia
+          component="img"
+          image={imageURL}
+          height="140"
+          alt="book img desc"
+        />
+        <Card sx={{ minWidth: '50vw', textAlign: 'left', display: 'flex' }}>
+          <CardContent>
+            <Typography gutterBottom variant="h3" sx={{ marginBottom: '0' }}>
+              {title}
+            </Typography>
+            <Typography variant="h5">
+              {authorName}
+              {' '}
+              |
+              {' '}
+              {illustratorName}
+              {' '}
+              (illustrator)
+            </Typography>
+            <Typography variant="body2">
+              {desc}
+            </Typography>
+          </CardContent>
+        </Card>
+
+      </div>
+    );
+
+    /* READ_ALOUD EMBED */
+    let readAloud = (<div />);
+    if (readAloudURL) {
+      /*
+        read_aloud_link is defined as a 'watch' URL,
+        so we must extract the embed code from this URL
+          WARNING: This code will break if the read_aloud_link
+          is not in the form of (anything)('watch?v=')(video id)$
+      */
+      const videoId = readAloudURL.split('watch?v=')[1];
+      readAloud = (
+        <div className="video-responsive">
+          <iframe
+            width="853"
+            height="480"
+            src={`https://www.youtube.com/embed/${videoId}`}
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+            title="Embedded youtube"
+          />
+        </div>
+      );
+    }
+
+    /* BOOKSHOP LINK, LinkUI from MaterialUI to not be confused with react-router */
+    let bookshop = <div />;
+    if (bookshopURL) {
+      bookshop = (
+        <LinkUI href={bookshopURL} rel="noreferrer" target="_blank">
+          Buy
+          {' '}
+          {title}
+          !
+        </LinkUI>
+      );
+    }
+
+    /* EDUCATOR GUIDE LINK */
+    let educators = <div />;
+    if (educatorURLs) {
+      educators = (
+        <List>
+          {educatorURLs.map((url) => (
+            <ListItem button component="a" href={url} key={url} alignItems="center">
+              <ListItemText sx={{ textAlign: 'center' }} primary={url} />
+            </ListItem>
+          ))}
+        </List>
+      );
+    }
+
+    return (
+      <Paper variant="outlined">
+        {synopsis}
+        {readAloud}
+        {bookshop}
+        <h4>Educator Guides</h4>
+        {educators}
+      </Paper>
```

Comment:
I think with the bookshop and educator links, maybe even the readAloud, you can rewrite this logic into the paper component using ternary operators! Again this could help with concision and error-handling, as if you rewrite the larger conditional, you can extend the ones I wrote to include any other conditions necessary to display the according link!

```suggestion
    /* EDUCATOR GUIDE LINK */
    return (
      <Paper variant="outlined">
        {synopsis /* change this into a component*/}
        {readAloud /* this could be a component or completed like shown below */}
        {bookshopURL && (
          <LinkUI href={bookshopURL} rel="noreferrer" target="_blank">
              Buy
              {' '}
              {title}
              !
          </LinkUI>)}
        <h4>Educator Guides</h4>
        {educatorURLs && (
          <List>
            {educatorURLs.map((url) => (
              <ListItem button component="a" href={url} key={url} alignItems="center">
                <ListItemText sx={{ textAlign: 'center' }} primary={url} />
              </ListItem>
            ))}
          </List>
          )}
        }
      </Paper>
```